### DISCUSSION TITLE
AtonStorage inverter support

### DISCUSSION THREAD

**Discussion by maatx** [slipx06/sunsynk-power-flow-card#310] [Aton Storage]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Hi, I like this card.
Is it possible to have native support for the AtonStorage inverter too? The (unofficial) Aton integration takes data from the cloud. I have attached the extracted entities (in a CSV file) and some screenshots

![Screenshot 2024-03-07 alle 11 32 37](https://github.com/slipx06/sunsynk-power-flow-card/assets/4578794/b196c173-ca3b-4099-876d-b12b4b308f86)
![Screenshot 2024-03-07 alle 11 32 50](https://github.com/slipx06/sunsynk-power-flow-card/assets/4578794/3f3e082b-49ae-4190-8840-c4ac0ee072bc)
![Screenshot 2024-03-07 alle 11 44 27](https://github.com/slipx06/sunsynk-power-flow-card/assets/4578794/da33aa11-eb6a-46fe-9f23-ecd1c74da780)
![Screenshot 2024-03-07 alle 11 44 39](https://github.com/slipx06/sunsynk-power-flow-card/assets/4578794/5b22f50b-86aa-4427-99c4-2ec5fc81e3ab)
[AtonStorage_entity.csv](https://github.com/slipx06/sunsynk-power-flow-card/files/14523085/AtonStorage_entity.csv)

### Expected behaviour

AtonStorage inverter integration

### Possible Solutions

If there is a configuration file with the entities to be submitted to the card, I can compile it myself to adapt it to the entities of the AtonStorage inverter

### Mode

Both

### Context / Reason

AtonStorage inverter integration

**Comment by slipx06** [comment]
Hi. Looks like most of the data is there. Have a look at some of the [examples ](https://slipx06.github.io/sunsynk-power-flow-card/index.html) in the documentation and try map your sensors.

**Comment by slipx06** [comment]
Moving this to the discussion section
